scale-tests/run.sh

```diff
@@ -0,0 +1,434 @@
+      --num-consumers-per-producer "${FAILING_NUM_CONSUMERS_PER_PRODUCER}" \
+      --producer-must-fail \
+      --producer-number-of-words "${FAILING_PRODUCER_NUMBER_OF_WORDS}" \
+      --producer-words-per-second "${FAILING_PRODUCER_WORDS_PER_SECOND}" \
+      --producer-spark-cores-max "${FAILING_PRODUCER_SPARK_CORES_MAX}" \
+      --producer-spark-executor-cores "${FAILING_PRODUCER_SPARK_EXECUTOR_CORES}" \
+      --consumer-must-fail \
+      --consumer-write-to-cassandra \
+      --consumer-batch-size-seconds "${FAILING_CONSUMER_BATCH_SIZE_SECONDS}" \
+      --consumer-spark-cores-max "${FAILING_CONSUMER_SPARK_CORES_MAX}" \
+      --consumer-spark-executor-cores "${FAILING_CONSUMER_SPARK_EXECUTOR_CORES}"
+
+  log 'Uploading failing jobs submissions file'
+  container_exec \
+    aws s3 cp --acl public-read \
+      "${FAILING_SUBMISSIONS_OUTPUT_FILE}" \
+      "s3://${TEST_S3_BUCKET}/${TEST_S3_FOLDER}/"
+else
+  log 'Skipping running of failing streaming jobs'
+fi
+
+if [ "${SHOULD_RUN_FINITE_STREAMING_JOBS}" = true ]; then
+  log 'Running finite jobs. Consumers write to Cassandra'
+  container_exec \
+    ./scale-tests/kafka_cassandra_streaming_test.py \
+      "${DISPATCHERS_JSON_OUTPUT_FILE}" \
+      "${INFRASTRUCTURE_OUTPUT_FILE}" \
+      "${FINITE_SUBMISSIONS_OUTPUT_FILE}" \
+      --jar "${TEST_ASSEMBLY_JAR_URL}" \
+      --num-producers-per-kafka "${FINITE_NUM_PRODUCERS_PER_KAFKA}" \
+      --num-consumers-per-producer "${FINITE_NUM_CONSUMERS_PER_PRODUCER}" \
+      --producer-number-of-words "${FINITE_PRODUCER_NUMBER_OF_WORDS}" \
+      --producer-words-per-second "${FINITE_PRODUCER_WORDS_PER_SECOND}" \
+      --producer-spark-cores-max "${FINITE_PRODUCER_SPARK_CORES_MAX}" \
+      --producer-spark-executor-cores "${FINITE_PRODUCER_SPARK_EXECUTOR_CORES}" \
+      --consumer-write-to-cassandra \
+      --consumer-batch-size-seconds "${FINITE_CONSUMER_BATCH_SIZE_SECONDS}" \
+      --consumer-spark-cores-max "${FINITE_CONSUMER_SPARK_CORES_MAX}" \
+      --consumer-spark-executor-cores "${FINITE_CONSUMER_SPARK_EXECUTOR_CORES}"
+
+  log 'Uploading finite jobs submissions file'
+  container_exec \
+    aws s3 cp --acl public-read \
+      "${FINITE_SUBMISSIONS_OUTPUT_FILE}" \
+      "s3://${TEST_S3_BUCKET}/${TEST_S3_FOLDER}/"
+else
+  log 'Skipping running of finite streaming jobs'
+fi
+
+if [ "${SHOULD_RUN_INFINITE_STREAMING_JOBS}" = true ]; then
+  log 'Running infinite jobs. Consumers do not write to Cassandra'
+  container_exec \
+    ./scale-tests/kafka_cassandra_streaming_test.py \
+      "${DISPATCHERS_JSON_OUTPUT_FILE}" \
+      "${INFRASTRUCTURE_OUTPUT_FILE}" \
+      "${INFINITE_SUBMISSIONS_OUTPUT_FILE}" \
+      --jar "${TEST_ASSEMBLY_JAR_URL}" \
+      --num-producers-per-kafka "${INFINITE_NUM_PRODUCERS_PER_KAFKA}" \
+      --num-consumers-per-producer "${INFINITE_NUM_CONSUMERS_PER_PRODUCER}" \
+      --producer-number-of-words "${INFINITE_PRODUCER_NUMBER_OF_WORDS}" \
```

Comment:
`INFINITE_PRODUCER_NUMBER_OF_WORDS` should *always* be 0.

Reply:
https://github.com/mesosphere/spark-build/pull/348/commits/5675d037074f934f4936177fef26f2bedf919150